### DISCUSSION TITLE
VSCode no intellisense and no typechecking in vue files

### DISCUSSION THREAD

**Discussion by amrap030** [vuejs/language-tools#1241] [Q&A] [answered]
Hello,

I installed Volar in a Vue 3 project using the script setup `lang="ts"` method in `.vue` files, but I don't get intellisense in my editor anymore. Normally in a `.ts` file I get some information when I hover over the code as follows:

<img width="616" alt="Bildschirmfoto 2022-04-27 um 13 44 42" src="https://user-images.githubusercontent.com/35061229/165511262-6ca79d17-1f41-4af5-b0e7-9944b7922f2d.png">

I also don't get any typechecking feedback in `.vue` files anymore.

Could anyone tell me why this happens or if I have to enable something?

**Comment by ktakdev** [comment, marked as answer]
In my case, updating VSCode to the latest version can solve the problem.

**Reply by amrap030** [reply]
Thanks, that fixed the issue :)
